src/io/flutter/view/FlutterView.java

```diff
@@ -135,7 +141,23 @@ void initToolWindow(ToolWindow window) {
       final AnAction sendFeedbackAction = new AnAction("Send Feedback", "Send Feedback", FlutterIcons.Feedback) {
         @Override
         public void actionPerformed(@NotNull AnActionEvent event) {
-          BrowserUtil.browse("https://goo.gl/WrMB43");
+          FlutterSdkVersion flutterSdkVersion = FlutterSdk.getFlutterSdk(myProject).getVersion();
+          String pluginVersion = PluginManager.getPlugin(FlutterUtils.getPluginId()).getVersion();
+          String ideVersion = String.format("%s %s", PlatformUtils.getPlatformPrefix(), ApplicationInfo.getInstance().getStrictVersion());
+
+          String flutterSdkVersionEntryId = "entry.1740350095";
+          String pluginVersionEntryId = "entry.1082356620";
+          String ideVersionEntryId = "entry.1842668120";
+
+          BrowserUtil.browse(
```

Comment:
I was going to comment on using `+` instead of `String.format` but in this case I think the StringFormat is a little clearer.